excluded_apps_further.txt

```diff
@@ -0,0 +1,14 @@
+Microsoft.WindowsVirtualDesktopBootloader
+Microsoft.WindowsSDK
+Microsoft.VCRedist.2019.arm64
+Google.Chrome
+Microsoft.Edge
+Microsoft.EdgeWebView2Runtime
+Microsoft.Office
+Microsoft.OneDrive
+Microsoft.Teams
+Mozilla.Firefox
+Mozilla.Firefox.DeveloperEdition
+Mozilla.Firefox.ESR
+TeamViewer.TeamViewer
+TeamViewer.TeamViewer.Host
```

Comment:
This is specific to your usage?

Reply:
Depends: the `Microsoft.WindowsVirtualDesktopBootloader` seems to be installed on all Windows 365 VMs. And I was not able to update it via WinGet (WinGet in general).
I have `Microsoft.WindowsSDK` on two different systems, and I’m unable to update it via WinGet.
A bit more special might be `Microsoft.VCRedist.2019.arm64`, same as above: WinGet find the update, install it, and again, and again, and again.
This is totally unrelated to your code! I have the same behavior with native WinGet! Therefore I thought it might be useful to share it with you and the community.
Could be something for the documentation and/or a Wiki page.

Reply:
I agree on that points. The thing is, we cannot have an exhaustive list of all "incompatible" apps. At the beginning this list was given as an example. But we can probably add apps that community report for WAU and then update the excluded_apps list.

Reply:
by the way, you can now use the new @KnifMelti feature `-ListPath` to link WAU to your own github `excluded-apps.txt`. This way, you can update your file in your repo and not publish in the community one that leave the default one as example. What do you think guys ? Don't know what's the best option

Reply:
You've needs to use the raw path then, so GitHub behaves as a normal web site

Reply:
> This is what I do. I have a plain text file on a public webserver (to use a short link). My point: Should we keep a List of Apps with known issues? Something like a Wiki within the Repository. Where we all can Contribute and explain why someone could or should exclude a package.

We could open a discussion ?!